src/test/java/uk/gov/hmcts/reform/iacaseapi/domain/handlers/presubmit/payment/MarkPaymentPaidPreparerTest.java

```diff
@@ -157,18 +154,16 @@ void should_return_error_for_old_pa_ea_hu_cases(AppealType appealType) {
         assertThat(returnedCallbackResponse.getErrors()).contains("You cannot mark this appeal as paid");
     }
 
-    @ParameterizedTest
-    @EnumSource(value = AppealType.class, names = { "PA", "EA", "HU", "EU" })
-    void should_return_error_for_old_pa_ea_hu_cases_when_is_lr_journey_and_dlrm_fee_remission_is_enabled(AppealType appealType) {
+    @Test
+    void should_return_error_for_old_pa_ea_hu_cases_when_is_lr_journey_and_dlrm_fee_remission_is_enabled() {
```

Comment:
Why did you remove the parametized test?
I case now this condition applies only to PA can you adapt the name of the test?

Reply:
Because it doesnt check anything except PA case on one of conditions because in the old condition (isEaHuEuAg && eaHuPaymentType.isEmpty()   **eaHuPaymentType** is not exists it is not populated anywhere. So we discussed with Rajesh to remove it.